demos/projects/PC/linux/main.c

```diff
@@ -241,6 +241,17 @@ static void prvSRand( UBaseType_t ulSeed )
 }
 /*-----------------------------------------------------------*/
 
+UBaseType_t uxRand( void )
```

Comment:
>uxRand [](http://example.com/codeflow?start=12&length=6)

weird, is this required?

Reply:
Yea if you enable `LOG_DEBUG` instead of just `LOG_INFO` it needs this function